recipes/templates/recipes/recipe/detail.html

```diff
@@ -79,6 +97,7 @@ <h2 class="text-3xl mt-6 mb-2">Ingredients</h2>
         </ul>
 </div>
 
+
 {% if recipe.complementary.exists %}
```

Comment:
gpt said this would issue another query, can you see that with debug toolbar?

it suggested:

```
{% with comps=recipe.complementary.all %}
{% if comps %}
  <div id="complementaryDishes"> ... {% for comp in comps %} ... {% endfor %}</div>
{% endif %}
{% endwith %}
```